api/v1/flow/flow.proto

```diff
@@ -64,6 +64,13 @@ message Flow {
     // destination_service contains the service name of the destination
     Service destination_service = 21;
 
+    // traffic_direction of the connection, e.g. ingress or egress
+    TrafficDirection traffic_direction = 22;
+
+    // policy_match_type is only applicable to the cilium event type PolicyVerdict
+    // https://github.com/cilium/cilium/blob/e831859b5cc336c6d964a6d35bbd34d1840e21b9/pkg/monitor/datapath_policy.go#L50
+    uint32 policy_match_type = 23;
```

Comment:
possibly this field needs to be an enum? otherwise the consumers of this API need to know what all the integers mean. not a blocker though.

Reply:
I choose u32 to match `drop_reason`, which is basically it's analog for dropped flows. In earlier versions I even called the field `forward_reason`, but that doesn't align with `flow.Verdict` (which is FORWARDED even for trace events which don't have a policy match)

Reply:
Oh, I have to correct myself. A policy verdict event is also emitted when the flow is about to be dropped. I will fix this!